### PR TITLE
Document running a DPS job programmatically

### PR DESCRIPTION
**Pull request by chuckwondo**

### CODE REVIEW

**Comment by wildintellect** [comment]
Inputs:
The input can be any geojson file, using a country boarder is just a shortcut some people might want to use. So we should move that section to optional, and add a section at the top on how to prepare a geojson file and get the s3 path to pass to the job.

Results:
We should describe the output format, that a gkpg file will be present in the dps_output path given in the results, and that it can be loaded with geopandas.

TODO: Look for a more user friendly way to parse the results of the DPS API. I've suggested https://pypi.org/project/xmltodict/ to UWG members before. Might have an example in icesat2_boreal

**Review by wildintellect — COMMENTED** [review]
Just a reminder we still need to update the instructions to use any geojson, the downloading of geojson is an example method but not necessarily the main method.

File: gedi-subset/README.md [hunk 1]
```diff
@@ -1,21 +1,118 @@
...
+
+```plain
+~/my-public-bucket/path/to/aoi.geojson
+```
+
+you would then supply the following URL as the `aoi` input value when running
+this algorithm as a DPS job, where `<USERNAME>` is your ADE username:
+
+```plain
+https://maap-ops-workspace.s3.amazonaws.com/shared/<USERNAME>/path/to/aoi.geojson
```
Comment (wildintellect) [inline gedi-subset/README.md:48]: Did you test to confirm this works? I can't recall if we need an https or s3 path for this (or if both work).

Comment (chuckwondo) [reply gedi-subset/README.md:48]: HTTPS works within the ADE when tested via `curl` in an ADE terminal window.

However, I did not directly test by attempting to run a job with such a URL as the `aoi` value (my jobs have always used the geoBoundaries public URLs).

I'll perform a quick test of both HTTPS and S3 as `aoi` input values to jobs (limited granules) to confirm/deny their validity.

File: gedi-subset/UseCaseTest.md [hunk 2]
```diff
@@ -233,3 +234,68 @@ Combined file size: 82,798,032 bytes
...
+architecture          x86_64
+machine_memory_size   30.36 GB
+directory_size        1511387136
+operating_system      CentOS
+job_start_time        2022-04-26T01:31:15.609852Z
+job_end_time          2022-04-26T02:27:35.695733Z
+job_duration_seconds  3380.085881
+```
+
+Oddly, this run indicated that 269 granule files were subsetted, not 266, so we
```
Comment (wildintellect) [inline gedi-subset/UseCaseTest.md:298]: GEDI is an ongoing mission, unless you date limit the query, new queries may pick up new data (which we want for now)

**Pull request merged**